team-telnyx/webrtc
# chore(WEBRTC-366): testing workflow

- Add cypress on `webrtc-test-release.yml` to run e2e tests

## 📝 To Do


## ✋ Manual testing

1. Go to `Actions` tab and choose the  `webrtc-test-release.yml`
2. Click em run workflow and see if you get green.

## 🦊 Browser testing

### Desktop
